### PR TITLE
Added maintainers file

### PR DESCRIPTION
**Pull request by sids-b**
**What type of PR is this?**
> /kind documentation

**What this PR does / why we need it**:
Adds maintainers list for kubeedge

**Special notes for your reviewer**:
Please check your name, emailId, and github handle. Thanks

### CODE REVIEW

File: MAINTAINERS [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+# Currently KubeEdge doesn't have folder wise maintainers.
+# 4th column will be added in the list to specify the folder maintained by an individual.
+#
+# Please keep the below list sorted in ascending order.
+
+Du Jun <dujun5@huawei.com> @m1093782566
+Li(Cindy) Xing <li.xing1@huawei.com> @CindyXing
+Rohit Sardesai <sardesai.rohit.prabhakar@huawei.com> @rohitsardesai83
+Siddharth Bhadri <siddharth.bhadri@huawei.com> @sids-b
+Wang(Kevin) Zefeng <wangzefeng@huawei.com> @kevin-wangzefeng
```
Comment (kevin-wangzefeng) [inline MAINTAINERS]: Zefeng (Kevin) Wang

File: MAINTAINERS [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+# Names added to this file should be in the following format:
+#     Individual's name <email address> @githubhandle
+#
+# Currently KubeEdge doesn't have folder wise maintainers.
+# 4th column will be added in the list to specify the folder maintained by an individual.
+#
+# Please keep the below list sorted in ascending order.
+
+Du Jun <dujun5@huawei.com> @m1093782566
+Li(Cindy) Xing <li.xing1@huawei.com> @CindyXing
```
Comment (kevin-wangzefeng) [inline MAINTAINERS]: Li (Cindy) Xing

File: MAINTAINERS [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+#
+# Names added to this file should be in the following format:
+#     Individual's name <email address> @githubhandle
+#
+# Currently KubeEdge doesn't have folder wise maintainers.
+# 4th column will be added in the list to specify the folder maintained by an individual.
+#
+# Please keep the below list sorted in ascending order.
+
+Du Jun <dujun5@huawei.com> @m1093782566
```
Comment (kevin-wangzefeng) [inline MAINTAINERS]: Jun Du

File: MAINTAINERS (hunk 1 above)
Comment (sids-b) [reply MAINTAINERS]: Done, thanks

File: MAINTAINERS (hunk 2 above)
Comment (sids-b) [reply MAINTAINERS]: Done, thanks

File: MAINTAINERS (hunk 3 above)
Comment (sids-b) [reply MAINTAINERS]: Done, thanks

**Pull request merged**